test/cache.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright 2019-2019 Ghent University
+#
+# This file is part of vsc-modules,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-modules
+#
+# vsc-modules is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# vsc-modules is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with vsc-modules.  If not, see <http://www.gnu.org/licenses/>.
+#
+'''
+
+From existing spiderT.lua:
+
+head -2000 spiderT.lua > test/data/spiderT.lua
+echo 'herehere' >> test/data/spiderT.lua
+tail -1000 spiderT.lua >> test/data/spiderT.lua
+
+and then some manual cleanup ;)
+    - insert 2nd mpath
+    - insert non-trivial default value
```

Comment:
these comments should be added to `test/data/spiderT.lua`?

Reply:
i'm not adding them to the lua file. if not here, i can add a Readme. to the test data dir

Reply:
Yeah, that would be better (the README bit)

Reply:
readme it is